spec/fixtures/files/minimal_planning_application.json

```diff
@@ -1,4 +1,4 @@
-{
+{ "application_type": 0,
```

Comment:
This works, due to how the `enum` works internally, but we are specifying in the swagger docs that we expect `application_type` to be "lawfulness_certificate"